tests/functional/test_plugins_multi_state_set_unset.py

```diff
@@ -54,22 +51,10 @@ def _prepare_multi_state(multi_state: List[str]) -> List[str]:
     return multi_state
 
 
-multi_state_comparator = build_objects_comparator(lambda obj: obj.multi_state, 'Multi state', _prepare_multi_state)
-
-check_cluster_related_objects_multi_state = build_objects_checker(
-    Cluster,
-    multi_state_comparator,
-    changed=MULTI_STATE,
-    unchanged=[],
-    allure_message='Check multi state of cluster related objects changed correctly',
-)
-
-check_provider_related_objects_multi_state = build_objects_checker(
-    Provider,
-    multi_state_comparator,
-    changed=MULTI_STATE,
-    unchanged=[],
-    allure_message='Check multi state of provider related objects changed correctly',
+check_objects_multi_state_changed = build_objects_checker(
+    field_name=FIELD_NAME,
+    changed=['ifeelgood!'],
+    extractor=(_extractor := lambda obj: _prepare_multi_state(obj.multi_state)),
```

Comment:
rename `_extractor` to `_multi_state_extractor` or something like

Reply:
Done